### ISSUE TITLE
Admin: reduce complexity of Publish block

### ISSUE THREAD

**Issue by ArthurClemens** [zotonic/zotonic#1353]
The idea of #1345 has exposed the grown complexity of the Publish block. I suggest to clean up:

* Separate Save and View actions
* Remove the action "Save and View". 
* Add separators, or move actions to separate blocks (not part of the floating Publish)

Something like:

```
Publish
[Save] [Cancel]
----------------------------------------------
[View v]
----------------------------------------------
More actions
[Delete] [Duplicate]
----------------------------------------------
Page status
[x] Published   [x] Prominent   [x] Protected   [x] Dependent
```

**Comment by mworrell** [comment]
The preview could also be made nice if we have a preview page that is live reloaded on every change.  At first this could be just the published version, if we have collaborative editing then we could show the edit version.

**Comment by ArthurClemens** [comment]
In that case a sticky top bar would be a replacement of the current side bar block.

```
-------------------------------------------------------------------
| Edit  |  Preview |                                [Save] [Cancel]
-       -----------------------------------------------------------
```

**Comment by ddeboer** [comment]
Good ideas: edit/preview should function like tabs, just like here on GitHub.

**Comment by mworrell** [comment]
Yes, why not. The floating block is a kind of hack anyway.
And in the top-bar we could add an `[ (Un)Publish ]` and `[ Options… ]` 

The `[ Options... ]` could open a dialog/popup with all the non-content meta data.
That would then clean up quite some panels and make the whole edit screen simpler.

**Comment by mworrell** [comment]
@ddeboer Not so sure about the tabs, as you will need to open the page in an iframe due to css/js

**Comment by arjan** [comment]
Yeah preview in a tab seems problematic to me as well unless you want to use an iframe.

And also take into account resources that don't have a page URL.

**Comment by ArthurClemens** [comment]
What is wrong with using an iframe to preview a page?

**Comment by mworrell** [comment]
It is problematic if we don't allow inclusion, so we should check the frame-options.
(Like the admin is not allowed to run inside an iframe because of security reasons)
